services/data-info/src/data_info/routes/tickets.clj

```diff
@@ -0,0 +1,45 @@
+(ns data-info.routes.tickets
+  (:use [common-swagger-api.schema]
+        [data-info.routes.domain.common]
+        [data-info.routes.domain.tickets])
+  (:require [data-info.services.tickets :as tickets]
+            [data-info.util.service :as svc]
+            [data-info.util.schema :as s]))
+
+
+(defroutes* ticket-routes
+
+  (context* "/tickets" []
```

Comment:
(I added this as a context because it might be useful/desirable to add operations which deal with individual tickets -- I could see a use in a `GET /tickets/:ticket-id` and `DELETE `/tickets/:ticket-id` at least. Those aren't in here, though, because I really just wanted to move the terrain endpoints for now and that's all bulk operations.